core/tests/protractor_desktop/extensions.js

```diff
@@ -102,6 +101,8 @@ describe('rich-text components', function() {
       // TODO(pranavsid98): This error is caused by the upgrade from Chrome 60
       // to Chrome 61. Chrome version at time of recording this is 61.0.3163.
       'chrome-extension://invalid/ - Failed to load resource: net::ERR_FAILED',
+      'The target origin provided (\'https://www.youtube.com\') does not ' +
+      'match the recipient window\'s origin (\'http://localhost:9001\').'
```

Comment:
Where does this come from? It looks like a same-origin policy block, which seems concerning

Reply:
https://stackoverflow.com/questions/27573017/failed-to-execute-postmessage-on-domwindow-https-www-youtube-com-http. It is a pretty common thing. For some weird reasons, happens on my laptop. Even happens on developer.google.com website. (The official website for youtube iframe api). This is pretty harmless IMO.

![Screenshot 2021-05-25 at 2 37 11 AM](https://user-images.githubusercontent.com/23002114/119407629-1e3f9600-bd02-11eb-9d41-59fba78c1d90.png)